target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -288,6 +288,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "3:lan:1" "5:lan:2" "4:wan"
 		;;
+	qca,ap143)
```

Comment:
Seems like this can be merged with case for comfast,cf-e560ac?

Reply:
Indeed, thanks.